core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/AkkaBakerConfig.scala

```diff
@@ -27,6 +28,17 @@ object AkkaBakerConfig extends LazyLogging {
 
   case class BakerValidationSettings(allowAddingRecipeWithoutRequiringInstances: Boolean)
 
+  case object ActorRecipeManagerType extends RecipeManagerType
+
+  case object InMemoryRecipeManagerType extends RecipeManagerType
+
+  sealed trait RecipeManagerType
+
+  implicit val recipeManagerTypeReader: ValueReader[RecipeManagerType] = (config: Config, path: String) => {
+    if (config.hasPath(path) && config.getString(path) == "actor") ActorRecipeManagerType
+    else InMemoryRecipeManagerType
```

Comment:
If we want to default the ActorManager impl we should also switch them arround here